src/components/layout/CommentInput.jsx

```diff
@@ -40,14 +49,16 @@ const CommentInput = () => {
             required
             onFocus={() => setFocus(true)}
             onChange={(event) => setInput(event.target.value)}
+            onKeyPress={submitCommentHandler}
           />
           {focus ? (
             <div className='comment-button-container'>
               <button
+                onClick={() => submitComment()}
                 className='submit-btn '
                 data-cy='comment-btn'
                 name='comment'
-                type='submit'>
+                type='button'>
```

Comment:
A button is a button, and you don't NEED to give it a type... I'm sorry to hear that you spent a lot of time on this.

Reply:
removed